bigframes/ml/utils.py

```diff
@@ -17,33 +17,41 @@
 
 import bigframes_vendored.constants as constants
 from google.cloud import bigquery
+import pandas as pd
 
 from bigframes.core import blocks
 import bigframes.pandas as bpd
 
 # Internal type alias
-ArrayType = Union[bpd.DataFrame, bpd.Series]
+InputArrayType = Union[bpd.DataFrame, bpd.Series, pd.DataFrame, pd.Series]
+OutputArrayType = Union[bpd.DataFrame, bpd.Series]
 
 
-def convert_to_dataframe(*input: ArrayType) -> Generator[bpd.DataFrame, None, None]:
+def convert_to_dataframe(
+    *input: InputArrayType,
+) -> Generator[bpd.DataFrame, None, None]:
     return (_convert_to_dataframe(frame) for frame in input)
 
 
-def _convert_to_dataframe(frame: ArrayType) -> bpd.DataFrame:
+def _convert_to_dataframe(frame: InputArrayType) -> bpd.DataFrame:
```

Comment:
Should we also handle array-like data like numpy arrays or even plain python list/tuples?

Reply:
Yeah, the effort should be trivial if we are to consolidate the conversion functions from the ml package to the core package. I can handle this in another CL as what is proposed above. Let me know your thoughts.